src/goto-cc/compile.cpp

```diff
@@ -80,8 +81,8 @@ bool compilet::doit()
   add_compiler_specific_defines(config);
 
   // Parse command line for source and object file names
-  for(std::size_t i=0; i<_cmdline.args.size(); i++)
-    if(add_input_file(_cmdline.args[i]))
+  for(std::size_t i = 0; i < cmdline.args.size(); i++)
```

Comment:
ranged for?

Reply:
Ok, will throw that in as separate commit.